spec/takeaway_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:takeaway){described_class.new(dish_klass, message_klass)}
+  let(:dish_klass){double(:dish_klass, :dishes=> [{ name: "Hamburger", price: 1},
+    {name: "Pizza", price: 2},
+    {name: "Hotdog", price: 2},
+    {name: "Tacos", price: 3},
+    {name: "Nachos", price: 1}
+  ])}
+  let(:message_klass){double(:message_klass)}
+  it 'lists available dishes and their prices' do
+    expect(takeaway.list_dishes).to eq("Hamburger: £1, Pizza: £2, Hotdog: £2, Tacos: £3, Nachos: £1")
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)